lib/dal-test/src/lib.rs

```diff
@@ -813,72 +819,15 @@ async fn migrate_local_builtins(
     info!("migrating intrinsic functions");
     func::migrate_intrinsics(&ctx).await?;
 
-    // FIXME(nick): restore builtin migration functionality for all variants.
-    info!("migrate minimal number of schemas for testing the new engine");
-
-    schema::migrate_pkg(&ctx, SI_DOCKER_IMAGE_PKG, None).await?;
-    schema::migrate_pkg(&ctx, SI_COREOS_PKG, None).await?;
-    schema::migrate_pkg(&ctx, SI_AWS_EC2_PKG, None).await?;
-    schemas::migrate_test_exclusive_schema_starfield(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_etoiles(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_morningstar(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_fallout(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_dummy_secret(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_swifty(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_katy_perry(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_pirate(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_pet_shop(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_validated_input(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_validated_output(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_bad_validations(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_large_odd_lego(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_large_even_lego(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_medium_even_lego(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_medium_odd_lego(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_small_odd_lego(&ctx).await?;
-    schemas::migrate_test_exclusive_schema_small_even_lego(&ctx).await?;
+    info!("migrating test exclusive schemas");
+    test_exclusive_schemas::migrate(&ctx).await?;
+
+    info!("migrations complete, commiting");
```

Comment:
Now that the new engine is functional, we can change the wording and code from its partially refactored state.